charts/pomerium/templates/_helpers.tpl

```diff
@@ -148,6 +167,23 @@ Adapted from : https://github.com/helm/charts/blob/master/stable/drone/templates
 {{- end -}}
 {{- end -}}
 
+{{/* Determine secret name for cache TLS Cert */}}
+{{- define "pomerium.cache.tlsSecret.name" -}}
+{{- if .Values.cache.existingTLSSecret -}}
+{{- .Values.cache.existingTLSSecret | trunc 63 | trimSuffix "-" -}}
+{{- /* TODO in future: Remove legacy logic */ -}}
```

Comment:
I think we should avoid carrying any legacy TODOs on the cache service at this point.  My thought was to remove the rest with the operator release.  No reason to add more debt ahead of that.

Reply:
👍